src/Controller/PayPalButtonsController.php

```diff
@@ -58,13 +59,13 @@ public function renderProductPageButtonsAction(Request $request): Response
 
         try {
             return new Response($this->twig->render('@SyliusPayPalPlugin/payFromProductPage.html.twig', [
+                'available_countries' => $this->availableCountriesProvider->provide(),
                 'clientId' => $this->payPalConfigurationProvider->getClientId($channel),
                 'completeUrl' => $this->router->generate('sylius_shop_checkout_complete'),
                 'createPayPalOrderFromProductUrl' => $this->router->generate('sylius_paypal_plugin_create_paypal_order_from_product', ['productId' => $productId]),
-                'processPayPalOrderUrl' => $this->router->generate('sylius_paypal_plugin_process_paypal_order'),
-                'locale' => $request->getLocale(),
                 'errorPayPalPaymentUrl' => $this->router->generate('sylius_paypal_plugin_payment_error'),
-                'available_countries' => $this->availableCountriesProvider->provide(),
+                'locale' => $request->getLocale(),
```

Comment:
Shouldn't it take `localeCode` from `localeContext` instead of directly from request? Is it even reliable source of locale code?